Core/GraphQL/GraphQLFiles/sell_fragment_fulfilmentPoint.graphql

```diff
@@ -0,0 +1,103 @@
+fragment FragmentFulfilmentPoint on FulfilmentPoint {
```

Comment:
The fragment names are not consistent with the existing fragments in sync with sell_fragment_paymentSourceDetails, canvas_fragment_widget and screenTranslation. 

Can we use camel-cased for the fragment name? And can we just name it `fulfilmentPoint` rather than `FragmentFulfilmentPoint`?

Reply:
will rename them once all changes are made 👍🏻

Reply:
`Cannot convert value of type 'ApolloType.Product.FulfilmentPoint.Type' to expected argument type 'GraphQLFragment.Type'`

We get these errors as it can't generate files with the same name as fragments, or rather it can, but can't tell which to use as part of the generation

Reply:
I understand the prefix Fragment is to avoid compiling error and there's a Apollo issue for this. https://github.com/apollographql/apollo-ios/issues/729 I'd suggest that let's unifying the fragment name after the issue is resolved